scripts/issue/967/deprecate_and_stop_emissions.py

```diff
@@ -0,0 +1,123 @@
+from brownie import Wei, interface, chain
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+from decimal import Decimal
+import time
+
+# addresses involved
+BIBBTCCRV = registry.eth.sett_vaults.bcrvIbBTC
+BRENCRV = registry.eth.sett_vaults.bcrvRenBTC
+LINK = registry.eth.treasury_tokens.LINK
+BADGER = registry.eth.treasury_tokens.BADGER
+
+# Empty schedule parameters
+START = 1668620976
+END = 1668620976
+DURATION = 0
+AMOUNT = 0
+INDEX = 39
+
+# Dripper and keeper constants
+UPKEEP_ID_TREE_22Q4 = 141  # https://automation.chain.link/mainnet/141
+
+# Tree BADGER deficit
+TREE_BADGER_DEFICIT = Wei(f"{33450.0339164155} ether")
+
+# Tree bcvxCRV BADGER deficit
+ORIGINAL_SCHEDULE_START = 1667606400
+ORIGINAL_SCHEDULE_END = 1672531200
+ORIGINAL_DURATION_WEEKS = Decimal(
+    (ORIGINAL_SCHEDULE_END - ORIGINAL_SCHEDULE_START) / 604800
+)
+TOTAL_AMOUNT = 12637714285714284415007
+AMOUNT_PER_WEEK = Wei(f"{1552} ether")
+REMAINING_WEEKS = Decimal((ORIGINAL_SCHEDULE_END - int(time.time())) / 604800)
```

Comment:
i think here is more precise to use `lastTimestamp` from the dripper when the upkeep was called last time, otherwise will be a gap of emissions which will not get filled which is from the last upkeep ts and the current ts

Reply:
Good call, fixing...

Reply:
nvm, it is accounted with the tree deficit 👍